members/members.css

```diff
@@ -0,0 +1,121 @@
+.title {
+    text-align: center;
+    margin-bottom: 2em;
+    font-size: 3.5em;
+}
+
+.sub-title {
+    font-size: 25px;
+    color: #444444;
+    position: relative;
+    padding-bottom: 1.25em;
+}
+
+.sub-title:after{
+    border-bottom: solid 1px;
+    bottom: 0;
+    content: '';
+    display: inline-block;
+    left: 0;
+    position: absolute;
+    width: 3rem;
+}
+
+
+h1 {
+    font-size: 30px;
+}
+
+h2{
+    font-size: 25px;
+}
+
+h3{
+    font-size: 20px;
+}
+
+h4{
+    font-size: 15px;
+}
+
+/* Five columns side by side */
+.column-card {
+  float: left;
+  width: 20%;
+  padding: 0 8px;
+  margin-bottom: 16px;
+}
+
+/* Display the columns below each other instead of side by side on small screens */
+@media screen and (max-width: 650px) {
+  .column-card {
+    width: 100%;
+    display: block;
+  }
+}
+
+/* Some left and right padding inside the container */
+.container-card {
+  padding: 0 16px;
+  padding-bottom: 16px;
```

Comment:
I think probably this can be minimize to just padding: 0 16px 16px 16px;

Reply:
Might need to update the comment above too